### DISCUSSION TITLE
XenOrchestra, XCP-NG

### DISCUSSION THREAD

**Discussion by BOAScripts** [selfhst/icons#135] [New Icon Request]
Hi, 
Here are the official repos:
- https://github.com/vatesfr
- https://github.com/vatesfr/xen-orchestra
- https://github.com/xcp-ng/xcp

**Comment by shollyethan** [comment]
Xen Orchestra and XCP-ng have been added - thanks!

**Reply by BOAScripts** [reply]
Many thanks
